home-manager.nix

```diff
@@ -332,16 +404,31 @@ in
                     fi
                 done
             fi
+
```

Comment:
Can we somehow make the above section DRYier? Not a big fan of copying and pasting the unmount stuff (although it's already duplicated on both `mkUnmount` and `mkBindMountService`.